rust-wasm-id-allocator/distributed-id-allocator/src/compressor/persistence.rs

```diff
@@ -27,6 +27,15 @@ pub enum DeserializationError {
     UnknownError,
 }
 
+impl ErrorEnum for DeserializationError {
+    fn get_error_string(&self) -> &'static str {
+        match self {
+            DeserializationError::PostcardError(_) => "Postcard error.",
```

Comment:
Can we get more info for this case? Does the postcard error implement any useful printing traits?

Reply:
It does. There's an issue with the non-static nature of that return value in the enum, combined with bindgen not allowing for returning references. I fiddled back and forth with it for a while, but if we feel it's valuable I can look into it further.

Reply:
I think it's worth it, considering that any versioning issues will actually surface as a postcard error (where an enum doesn't have a member postcard thinks it should) so any compat errors would get bucketed into "Postcard error."

Reply:
Pushed a change. Because our ErrorEnum trait has get_error_string templated to return a static, I removed the trait implementation and instead implemented get_error_string directly. Lmk if this doesn't work for us.

Reply:
Sounds good!